### DISCUSSION TITLE
Is support Pion to Pion when ice udp mux mode ?

### DISCUSSION THREAD

**Discussion by ziminghua** [pion/webrtc#2598] [Q&A] [answered]
I very like pion. I use pion to pion when udp mux mode,  but I find the udp packets can't right dispatch. I has more than one peerconnection. I need other config ?

I read the code , dispatch by host ip and port, but when udp mux mode the port is my config, so dispatch is error.

**Comment by Sean-Der** [comment, marked as answer]
Hey @ziminghua 

Unfortunately that will not work. UDPMux identifies the remote by its unique IP+Port. If both sides are using a `UDPMux` you have no way to uniquely identify them.

You can still uniquely identify the traffic though! It will take some effort to make it work (if you care). Happy to help with this!

* DTLS can be demuxed via ConnectionID
* RTP can be demuxed via SSRC
